docs/user_docs/observability/monitor-database.md

```diff
@@ -245,100 +210,64 @@ componentSpecs:
+    disableExporter: true # Set to `false` to enable exporter
 ...
-spec:
-   ...
-   componentSpecs:
-   ...
-      disableExporter: false
-```
-
-If `disableExporter: false` is not shown in the output, it means the monitoring function of this cluster is not enabled and you need to enable it first.
-
-```bash
-kbcli cluster update mycluster --disable-exporter=false
 ```
 
 </TabItem>
 
 </Tabs>
 
-### View the dashboard
-
-You can view the dashboard of the corresponding cluster via Grafana Web Console. For more detailed information, see the [Grafana dashboard documentation](https://grafana.com/docs/grafana/latest/dashboards/).
+When the cluster is running, each Pod should have a sidecar container, named `exporter` running the postgres-exporter.
 
-### (Optional) Enable remote write
+### Create PodMonitor
 
-Remote write is an optional step and you can enable it based on your actual needs. KubeBlocks provides an Addon, `victoria-metrics-agent`, to push the monitoring data to a third-party monitoring system compatible with the Prometheus Remote Write protocol. Compared with the native Prometheus, [vmgent](https://docs.victoriametrics.com/vmagent.html) is lighter and supports the horizontal extension. For detailed settings, you can refer to [Victoria Metrics docs](https://artifacthub.io/packages/helm/victoriametrics/victoria-metrics-agent).
+1. Query `scrapePath` and `scrapePort`.
 
-<Tabs>
+   Retrieve the `scrapePath` and `scrapePort` from the Pod's exporter container.
 
-<TabItem value="kubectl" label="kubectl" default>
-
-Install the `victoria-metrics-agent` Addon.
+   ```bash
+   kubectl get po mycluster-postgresql-0 -oyaml | yq '.spec.containers[] | select(.name=="exporter") | .ports '
+   ```
 
-```bash
-helm install vm kubeblocks/victoria-metrics-agent --set remoteWriteUrls={http://<remoteWriteUrl>:<port>/<remote write path>}
-```
+   <details>
 
-</TabItem>
+   <summary>Expected Output</summary>
 
-<TabItem value="kbcli" label="kbcli">
+   ```bash
+   - containerPort: 9187
+     name: http-metrics
+     protocol: TCP
+   ```
 
-1. Enable data push.
+   </details>
 
-   You just need to provide the endpoint which supports the Prometheus Remote Write protocol and multiple endpoints can be supported. Refer to the tutorials of your third-party monitoring system for how to get an endpoint.
+2. Create `PodMonitor`.
 
-   The following examples show how to enable data push by different options.
+   Apply the `PodMonitor` file to monitor the cluster. You can find the example YAML file in the [KubeBlocks Addons repo](https://github.com/apecloud/kubeblocks-addons/blob/main/examples/postgresql/pod-monitor.yml).
 
    ```bash
-   # The default option. You only need to provide an endpoint with no verification.
-   # Endpoint example: http://localhost:8428/api/v1/write
-   kbcli addon enable victoria-metrics-agent --set remoteWriteUrls={http://<remoteWriteUrl>:<port>/<remote write path>}
+   kubectl apply -f examples/postgresql/pod-monitor.yaml
```

Comment:
show the content of the file ,  instead of a path, cannot be located from the context.